airflow/utils/context.py

```diff
@@ -59,6 +59,7 @@
     "inlets",
     "logical_date",
     "macros",
+    "map_index_template",
```

Comment:
I don’t think we need this field in the context.

Reply:
If [this](https://github.com/apache/airflow/pull/36797#discussion_r1467173302) is the desired implementation, then I think we do need it here?